packages/vaadin-lumo-styles/components.css

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright (c) 2017 - 2025 Vaadin Ltd.
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
```

Comment:
Accidental?

Reply:
Nope, it's an auto-generated file, so I think it's better to stick to the license header that ESLint adds automatically.